### PR TITLE
Spellset

### PR DESCRIPTION
**Pull request by luciensadi**

### CODE REVIEW

File: src/constants.cpp [hunk 1]
```diff
@@ -1660,6 +1660,17 @@ const char *attributes[] =
     "Reaction"
   };
 
+const char *short_attributes[] =
```
Comment (luciensadi) [inline src/constants.cpp]: Used to confirm which attribute the imm wants affected by the attribute spells in spellset.

File: src/interpreter.cpp [hunk 2]
```diff
@@ -864,6 +865,7 @@ struct command_info cmd_info[] =
     { "spool"    , POS_DEAD    , do_spool    , 0, 0, FREE }, 
     { "speed"    , POS_RESTING , do_speed    , 0, 0 },
     { "spells"   , POS_SLEEPING, do_spells   , 0, 0 },
+    { "spellset" , POS_SLEEPING, do_spellset , LVL_DEVELOPER, 0 },
```
Comment (luciensadi) [inline src/interpreter.cpp]: Should this require higher privs? I don't recall what 'developer' is numerically.

Comment (jalles76) [inline src/interpreter.cpp]: skillset uses LVL_DEVELOPER so I think it's fine.

**Comment by luciensadi** [comment]
Will merge this for now and file an issue to improve spellset.

**Pull request merged**